extensions/stackdriver/edges/edge_reporter.cc

```diff
@@ -155,11 +155,10 @@ void EdgeReporter::addEdge(const ::Wasm::Common::RequestInfo& request_info,
   edge->mutable_destination()->CopyFrom(node_instance_);
 
   auto protocol = request_info.request_protocol;
-  if (protocol == "http" || protocol == "HTTP") {
+  // TODO: add support for HTTPS
```

Comment:
what is the plan here?

Reply:
I haven't seen it being set to HTTPS or https anywhere, so it was dead code. I placed TODO to actually implement it.